docs/abi-spec.rst

```diff
@@ -310,6 +312,7 @@ An event description is a JSON object with fairly similar fields:
 - `inputs`: an array of objects, each of which contains:
   * `name`: the name of the parameter;
   * `type`: the canonical type of the parameter.
+  * `components`: used for tuple types (more below).
```

Comment:
If you with `struct` then I'd add a comment here to mean `tuple` or `struct` types.